lib/renderer.js

```diff
@@ -198,11 +198,15 @@
      * Either expand to allow all dimensions or have a fixed limit base on a config flag
      *
      * @param {Bounds} bounds
+     * @param {number} scaleX
+     * @param {number} scaleY
      * @return {number}
      */
-    BaseRenderer.prototype._getPixmapMaxDimensions = function (bounds) {
+    BaseRenderer.prototype._getPixmapMaxDimensions = function (bounds, scaleX, scaleY) {
```

Comment:
we already shipped with it plural and the config settings is ```config["expand-max-dimensions"]```. We do look at both dimensions so maybe that counts.